### DISCUSSION TITLE
visual studio code使用疑問

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#344] [Q&A]
使用visual studio code的時候，沒有辦法直接在終端機執行程式。沒有辦法直接執行整個*.py，也沒有辦法執行如print(2)這種指令。不知道老師或同學熟不熟悉visual studio code終端機的操作。
<img width="960" alt="VS code終端機" src="https://user-images.githubusercontent.com/86505454/149627915-1e04709c-616a-42ee-84b3-9779958eee32.png">

**Comment by lx3211** [comment]
你要先開啟python3
在terminal打python3進入環境後才能輸入指令喔
<img width="496" alt="截圖 2022-02-21 01 07 29" src="https://user-images.githubusercontent.com/96198440/154854795-c0915820-13ea-4fd4-8009-27ffa99cc64e.png">
開啟後輸入的地方應該要是>>>

**Comment by handsomeniko** [comment]
你點一下最左邊那排倒數第二個，看看python裝了沒
